wallet/src/de/schildbach/wallet/AppDatabase.java

```diff
@@ -5,27 +5,38 @@
 import androidx.room.RoomDatabase;
 import androidx.room.TypeConverters;
 
+import de.schildbach.wallet.data.BlockchainIdentityData;
+import de.schildbach.wallet.data.BlockchainIdentityDataDao;
+import de.schildbach.wallet.data.BlockchainIdentityDataDaoAsync;
 import de.schildbach.wallet.data.BlockchainState;
 import de.schildbach.wallet.data.BlockchainStateDao;
 import de.schildbach.wallet.data.IdentityCreationState;
 import de.schildbach.wallet.data.IdentityCreationStateDao;
+import de.schildbach.wallet.data.IdentityCreationStateDaoAsync;
 import de.schildbach.wallet.data.RoomConverters;
 import de.schildbach.wallet.rates.ExchangeRate;
 import de.schildbach.wallet.rates.ExchangeRatesDao;
 
 /**
  * @author Samuel Barbosa
  */
-@Database(entities = {ExchangeRate.class, BlockchainState.class, IdentityCreationState.class}, version = 5)
+@Database(entities = {ExchangeRate.class, BlockchainState.class, IdentityCreationState.class, BlockchainIdentityData.class}, version = 7)
```

Comment:
Version 6 was in a previous commit in this PR.  Since we are not releasing any of these apk's into production, is there need to keep incrementing the version number when we make changes.  The master branch is still on Version 2.  Couldn't we keep this version at 3?  or would we do that later? or does it not matter?

Reply:
Lowered to 6:  https://github.com/dashevo/dash-wallet/pull/393/commits/66c8716a0809b9c5bcd6d5c1315afa490d260416

Reply:
Do we need to commit the 6.json file also?  It is not included in this PR.

Reply:
No, it shouldn't be added to VCS it is generated automatically by Room.